advanced/readme_kiali.adoc

```diff
@@ -0,0 +1,183 @@
+grafana                grafana-istio-system.192.168.15.12.nip.io                          grafana                http                            None
+istio-ingress          istio-ingress-istio-system.192.168.15.12.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.192.168.15.12.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.192.168.15.12.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.192.168.15.12.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.192.168.15.12.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.192.168.15.12.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (in my case,
+"kiali-istio-system.192.168.15.12.nip.io"), so let's access it:
+
+image::https://i.imgur.com/rG5tdZu.png[Login Page]
+
+This is the login page. The default credentials is "admin/admin", but it's
+recommended to change it before using it in production.
+
+== Service Graph
+
+After login, you should be greeted with the graph page:
+
+image::https://i.imgur.com/PtKVfe4.png[Graph Page]
+
+It shows a graph with all the microservices, connected by the requests going
+through then. On this page, you can see how the services interact with each
+other.
+
+== Service Listing
+
+In this page, you can get a listing of all the services that are running on the
+cluster, and get some information on them, such as health status:
+
+image::https://i.imgur.com/Gytp5Wm.png[Service Listing Page]
+
+Clicking on a service you'll see this page, where you can introspect into more
+information on the service:
+
+image::https://i.imgur.com/msQjCb4.png[Service Details]
+
+You can see the health status of the service (a Healthy service means one that
+is is up and responding correctly):
+
+image::https://i.imgur.com/MVpqDx8.png[Health Status]
+
+You can also see the deployments:
+
+image::https://i.imgur.com/TdyjCOu.png[Deployments]
+
+The source services:
+
+image::https://i.imgur.com/Qof6a8s.png[Source Services]
+
+And the virtual services on the istio infrastructure:
+
+image::https://i.imgur.com/mlZknRv.png[Virtual Services]
+
+== Istio Config
+
+In this page, you can see all currently running config rules, such as Virtual
+Services, Route Rules, Routes, Circuit Breakers, Fault Injection and so on.
+
+image::https://i.imgur.com/KWhZ29z.png[Istio Config Page]
+
+== Distributed Tracing
+
+In this page you can see the distributed tracing, as provided by Jaeger.
+
+image::https://i.imgur.com/hY78haT.png[Distributed Tracing Page]
+
+== Generating Sample Data
```

Comment:
I would move this section about Bookinfo to up before you have the user log in.  That way they'll have some example data to view when they read all the descriptions of the pages.